src/Sarif/NotYetAutoGenerated/AddressComparer.cs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// Defines methods to support the comparison of objects of type Address for sorting.
+    /// </summary>
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "1.1.4.0")]
+    internal sealed class AddressComparer : IComparer<Address>
+    {
+        internal static readonly AddressComparer Instance = new AddressComparer();
+
+        public int Compare(Address left, Address right)
+        {
+            int compareResult = 0;
+
+            // TryReferenceCompares is an autogenerated extension method
+            // that will properly handle the case when 'left' is null.
+            if (left.TryReferenceCompares(right, out compareResult))
+            {
+                return compareResult;
+            }
+
+            compareResult = left.AbsoluteAddress.CompareTo(right.AbsoluteAddress);
+            if (compareResult != 0)
+            {
+                return compareResult;
+            }
+
+            if (left.RelativeAddress.TryReferenceCompares(right.RelativeAddress, out compareResult))
+            {
+                return compareResult;
+            }
+
+            compareResult = left.RelativeAddress.Value.CompareTo(right.RelativeAddress.Value);
```

Comment:
>left.RelativeAddress.Value [](http://example.com/codeflow?start=28&length=26)

I think this will generate a null reference exception when RelativeAddress is null.

Can you add some tests as:
left.RelativeAddress = right.relativeAddress = null
left.RelativeAddres = null, right.relativeAddress=  1
left.RelativeAddress = 1, right.relativeAddress = null
left.relativeAddress = 1, right.relativeaddress = 1